scipy/special/special/hyp2f1.h

```diff
@@ -49,6 +49,8 @@
 #include "cephes/poch.h"
 #include "digamma.h"
 
+extern "C" double cephes_hyp2f1(double, double, double, double);
+
```

Comment:
I don't like having this `extern "C"` from outside cephes thrown into what should be a self contained library. I think it's fine to just copy the `hyp2f1` from here into your PR.

https://github.com/scipy/scipy/blob/820ed56e237cd7091a2c4d4a5b5f89a7ef993975/scipy/special/special/cephes/hyp2f1.h

Reply:
Done.